src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -9181,23 +9181,27 @@ private ForStatementSyntax ParseForStatement(SyntaxList<AttributeListSyntax> att
             var (variableDeclaration, initializers) = eatVariableDeclarationOrInitializers();
 
             // Pulled out as we need to track this when parsing incrementors to place skipped tokens.
```

Comment:
It looks like this comment was specific to `secondSemicolonToken` before, is it still relevant now?

Reply:
yes/  will move.